server/routes/message/post-group-message.js

```diff
@@ -13,7 +13,7 @@ router.post('/:id/message', authenticateToken, async (req, res) => {
     }
 
     try {
-        const message = new Message(id,msg,groupId);
+        const message = new Message(id,groupId,msg);
```

Comment:
Let's try to stay away from abbreviations.  `msg` could be renamed to `text` or `content`. 
```suggestion
        const message = new Message(id,groupId,text);
```